### PR TITLE
fix: link to newrelic_entity

### PR DESCRIPTION
**Pull request by knapo**

### CODE REVIEW

**Comment by mbazhlekova** [comment]
@knapo Thank you for your PR! One thing, can you reword your commit message to follow conventional commits? Like this `fix: link to newrelic_entity`

**Comment by knapo** [comment]
@mbazhlekova done!

**Comment by knapo** [comment]
@mbazhlekova Looks like it's not easily fixable, since it either works on GH or terraform docs page, because the link structure is different. Is there any trick to make it working on both? Similar case is with `additional requirements ` link on https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/entity.

File: website/docs/d/application.html.markdown [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: |-
 ---
 
 # Data Source: newrelic\_application
-~> **DEPRECATED** Use at your own risk. Use the [`newrelic_entity`](/docs/providers/newrelic/d/entity.html) data source instead. This feature may be removed in the next major release.
+~> **DEPRECATED** Use at your own risk. Use the [`newrelic_entity`](../../docs/data-sources/entity) data source instead. This feature may be removed in the next major release.
```
Comment (antonblr) [inline website/docs/d/application.html.markdown:10]:
```suggestion
~> **DEPRECATED** Use at your own risk. Use the [`newrelic_entity`](/providers/newrelic/newrelic/latest/docs/data-sources/entity) data source instead. This feature may be removed in the next major release.
```

I'd use absolute url (`https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/entity`), but in sake of consistency (e.g. #1659) you could omit `https://registry.terraform.io` part.

**Pull request closed without merging**